packages/lodestar/src/network/gossip/handlers/index.ts

```diff
@@ -111,7 +111,7 @@ export function getGossipHandlers(modules: ValidatorFnsModules, options: GossipH
 
       // `validProposerSignature = true`, in gossip validation the proposer signature is checked
       chain
-        .processBlock(signedBlock, {validProposerSignature: true})
+        .processBlock(signedBlock, {validProposerSignature: true, blsVerifyMainThread: true})
```

Comment:
Add explanation above the `blsVerifyMainThread` option linking to the issue and investigation that motivated this change